README.md

```diff
@@ -29,7 +29,8 @@ Machine, which is required to run Docker on Mac/Windows hosts.
 2. Build the container(s)
 
    ```
-   $ docker-compose build
+   $ docker-compose -f conf/docker/docker-compose.yml build
+   $ npm install
    ```
 
 3. Configure environment variables
```

Comment:
this needs to be for all 3 sample.env files, and it's sample.env, not env.sample

Reply:
Think I got it!